README.md

```diff
@@ -79,15 +79,19 @@ make testacc TESTARGS='-run=TestAccXXX'
 
 To contribute, please read the contribution guidelines: [Contributing to Terraform - Anexia Cloud Provider](CONTRIBUTING.md)
 
+1. Make your code changes on the branch of your fork of [terraform-provider-anxcloud](https://github.com/anexia-it/terraform-provider-anxcloud)
+2. Create a pull request
+3. Add your changes to the **Unreleased** section in [CHANGEME.md](CHANGEME.md)
+4. Trigger the integration tests via a `/ok-to-test sha=$SHA` comment. The $SHA represents the last commit in the PR.
```

Comment:
outside contributors cannot trigger tests, we should clarify that here (and should finally move to Github Workflow approvals instead of ok-to-test... but not this PR)